ext/ftp/ftp.stub.php

```diff
@@ -58,7 +58,9 @@
      * @cvalue PHP_FTP_MOREDATA
      */
     const FTP_MOREDATA = UNKNOWN;
+}
 
+namespace FTP {
```

Comment:
Then the FTP namespace should be declared after the global one so that there's no need for repeating `namespace {`